transaction/src/manifest/parser.rs

```diff
@@ -282,19 +282,30 @@ impl Parser {
             TokenKind::List => self.parse_list(),
             TokenKind::Set => self.parse_set(),
             TokenKind::Map => self.parse_map(),
-            TokenKind::Decimal
-            | TokenKind::PreciseDecimal
-            | TokenKind::PackageAddress
-            | TokenKind::SystemAddress
-            | TokenKind::ComponentAddress
-            | TokenKind::ResourceAddress
-            | TokenKind::Hash
-            | TokenKind::Bucket
-            | TokenKind::Proof
-            | TokenKind::NonFungibleId
-            | TokenKind::NonFungibleAddress
-            | TokenKind::Expression
-            | TokenKind::Blob => self.parse_scrypto_types(),
+            /* Global address */
+            TokenKind:: PackageAddress |
```

Comment:
There's inconsistent spacing after the :: here, I assume not intentional? :)

Reply:
For some reason, the formatter doesn't work for this blob of code. Will make them consistent.